bitcoin/src/psbt/error.rs

```diff
@@ -72,6 +72,8 @@ pub enum Error {
     /// Conflicting data during combine procedure:
     /// global extended public key has inconsistent key sources
     CombineInconsistentKeySources(Box<ExtendedPubKey>),
+    /// Parsing error.
+    ParseFailed(&'static str),
```

Comment:
Why a bunch of strings instead of separate variants for each, carrying more error information?

Reply:
This will become separate variants like in #1310 when @arturomf94 follows this up. This PR only fixes the first two bullet points of the PSBT Error untangling

Reply:
Awesome!